### ISSUE TITLE
Android error when lastKnownBatteryLevel is null

### ISSUE THREAD

**Issue by vanlooverenkoen** [DarkerWaters/flic_button#1]
First of all thanks for your awesome library. I have been waiting for this plugin for ages :D (I don't have iOS swift/ objective -c experience) 

I have an issue when fetching all the buttons on android. I will try to figure out why the `getLastKnownBatteryLevel` is null

```
E/MethodChannel#flic_button( 7757): Failed to handle method call
E/MethodChannel#flic_button( 7757): java.lang.NullPointerException: Attempt to invoke virtual method 'int io.flic.flic2libandroid.BatteryLevel.getEstimatedPercentage()' on a null object reference
E/MethodChannel#flic_button( 7757): 	at uk.co.darkerwaters.flic_button.FlicButtonPlugin.ButtonToJson(FlicButtonPlugin.java:118)
E/MethodChannel#flic_button( 7757): 	at uk.co.darkerwaters.flic_button.FlicButtonPlugin.onMethodCall(FlicButtonPlugin.java:183)
E/MethodChannel#flic_button( 7757): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:233)
E/MethodChannel#flic_button( 7757): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/MethodChannel#flic_button( 7757): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:818)
E/MethodChannel#flic_button( 7757): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/MethodChannel#flic_button( 7757): 	at android.os.MessageQueue.next(MessageQueue.java:335)
E/MethodChannel#flic_button( 7757): 	at android.os.Looper.loop(Looper.java:183)
E/MethodChannel#flic_button( 7757): 	at android.app.ActivityThread.main(ActivityThread.java:7660)
E/MethodChannel#flic_button( 7757): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#flic_button( 7757): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
E/MethodChannel#flic_button( 7757): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
```

Haha quite funny that I just found your repo `scorepall`. Apparently we had exactly the same idea to count our points. In racket sports. (I am working on a squash/padel/tennis/badminton app)

**Comment by butterbrain** [comment]
Thanks Van

My objective-c isn't great as I had to learn it to do what I did. Well done
for picking up on my "null" - don't know what I was thinking. As you could
maybe tell, I wrote it without null safety too so thanks for fixing that
too. I merged the branch and uploaded a new version 0.1.6 so upload and
check it out. Hopefully it works.

Doug

**Comment by vanlooverenkoen** [comment]
I already used my custom git dependency in my pubspec.yaml. And everything works fine now. I will update to 0.1.6 tonight.
